Sources/SwiftSyntax/Trivia.swift.gyb

```diff
@@ -99,19 +99,24 @@ public struct Trivia {
   public var byteSize: Int {
     return sourceLength.utf8Length
   }
-
 % for trivia in TRIVIAS:
+
 %   if trivia.is_collection():
 %   joined = ''.join(trivia.swift_characters)
-    /// Return a piece of trivia for some number of '${joined}' characters.
-    public static func ${trivia.lower_name}s(_ count: Int) -> Trivia {
-      return [.${trivia.lower_name}s(count)]
-    }
+  /// Returns a piece of trivia for some number of '${joined}' characters.
+  public static func ${trivia.lower_name}s(_ count: Int) -> Trivia {
+    return [.${trivia.lower_name}s(count)]
+  }
+
+  /// Gets a piece of trivia for '${joined}' characters.
+  public static var ${trivia.lower_name}: Trivia {
+    return [.${trivia.lower_name}s(1)]
```

Comment:
I don’t even think we need the `[` and `]`. Without those we’re just calling into the function defined above.